junit-test/output-jvm/src/test/scala/scala/scalanative/junit/utils/JUnitTestPlatformImpl.scala

```diff
@@ -25,9 +24,18 @@ object JUnitTestPlatformImpl {
     }
   }
 
-  def writeLines(lines: List[String], file: String): Unit =
-    Files.write(Paths.get(file), lines.asJava, UTF_8)
+  def writeLines(lines: List[String], file: String): Unit = {
+    val jLines = new LinkedList[String]()
+    lines.foreach(jLines.add)
+    Files.write(Paths.get(file), jLines, UTF_8)
+  }
 
-  def readLines(file: String): List[String] =
-    Files.readAllLines(Paths.get(file), UTF_8).asScala.toList
+  def readLines(file: String): List[String] = {
+    val buf = new UnrolledBuffer[String]()
+    val it  = Files.readAllLines(Paths.get(file), UTF_8).iterator()
+    while (it.hasNext) {
+      buf += it.next()
+    }
+    buf.toList
```

Comment:
```suggestion
    val builder = List.newBuilder[String]
    val it  = Files.readAllLines(Paths.get(file), UTF_8).iterator()
    while (it.hasNext) {
      builder += it.next()
    }
    builder.result()
```
which is much more efficient than using another data structure.